roles/apt/templates/30apt-proxy.j2

```diff
@@ -1,2 +1,2 @@
 // {{ ansible_managed }}
-Acquire::http::Proxy "{{ apt_proxy }}";
+Acquire::http::Proxy "http://{{ inventory_hostname }}:8000/";
```

Comment:
The file doesn't exit if there is no proxy 

 38 - name: setup apt proxy
 39   template:
 40     src: templates/30apt-proxy.j2
 41     dest: /etc/apt/apt.conf.d/30apt-proxy
 42   when: apt_proxy != ""